dir.traversal.targets

```diff
@@ -112,6 +112,34 @@
     <Message Text="%(PackageIds.Identity)" Importance="High" />
   </Target>
 
+  <Target Name="GetTargetPath" DependsOnTargets="FilterProjects" Returns="@(_ProjectTargetPath)">
+    <SelectTargetPath TargetPaths="@(Project)">
+      <Output TaskParameter="TargetPath" ItemName="_TargetProject"/>
+    </SelectTargetPath>
+    <MSBuild Targets="GetTargetPath"
```

Comment:
Why does this need to be in the traversal itself? Why cann't you just call this target on all the projects in index.proj directly? That would enable you to just let msbuild append all of them for you.